attestation-agent/attester/src/az_snp_vtpm/mod.rs

```diff
@@ -31,7 +31,7 @@ struct Evidence {
 
 #[async_trait::async_trait]
 impl Attester for AzSnpVtpmAttester {
-    async fn get_evidence(&self, report_data: Vec<u8>) -> anyhow::Result<String> {
+    async fn evidence(&self, report_data: Vec<u8>) -> anyhow::Result<String> {
```

Comment:
Is this change necessary?imo that Rust does not use getter naming, which is for structure fields, and attester's `get_evidence` is more like a procedure call, so I think it looks okay. wdyt?

Reply:
I can drop this. I am not totally convinced but doesn't really matter.